www/src/pages/portfolio/index.tsx

```diff
@@ -10,7 +10,13 @@ const Index = () => (
       Whoops, page is WIP!
     </Center>
     <Center as={Heading.h2} mt="5" align="center">
-      Howsa 'bout going {` `} <Link href="/home">home</Link>?
+      <span>
+        Howsa 'bout going {` `}
+        <Link href="/">
+          <Heading.h2 display="inline">home</Heading.h2>
```

Comment:
Technically, having h tags in a tags is invalid in specs other than html5...

Reply:
Look into 'inheriting' fontsizes for link tags? Might just need to explicitly set them... so shorthands/ aliases for font sizes might come in handy.